sw/device/tests/spi_host_config_test.c

```diff
@@ -57,8 +61,8 @@ bool test_main(void) {
 
 static status_t spi_config_test(dif_spi_host_t *spi) {
   dif_spi_host_config_t config = {
-      .spi_clock = 40000,
-      .peripheral_clock_freq_hz = (uint32_t)kClockFreqUsbHz,
+      .spi_clock = kSpiClock,
+      .peripheral_clock_freq_hz = (uint32_t)kClockFreqHiSpeedPeripheralHz,
```

Comment:
I think this should be
```
.peripheral_clock_freq_hz = (uint32_t)kClockFreqHiSpeedPeripheralHz / 2,
```
like in `sw/device/tests/power_virus_systemtest.c`.

Reply:
Ah, actually I am not sure, because on FPGA `kClockFreqHiSpeedPeripheralHz` is 24 MHz rather than 96 MHz.

Reply:
The `kClockFreqUsbHz` constant on the FPGA is greater than the `kClockFreqCpuMhz`. Is that right?
If following ratio of Silicon, the first should be 12MHz, and therefore `.peripheral_clock_freq_hz = (uint32_t)kClockFreqHiSpeedPeripheralHz / 2,` would be correct.

Reply:
> The `kClockFreqUsbHz` constant on the FPGA is greater than the `kClockFreqCpuMhz`. Is that right?

Yes that's right. The USB clock must always be 48 MHz but Ibex only runs at 24 MHz on the FPGA.

> If following ratio of Silicon, the first should be 12MHz, and therefore `.peripheral_clock_freq_hz = (uint32_t)kClockFreqHiSpeedPeripheralHz / 2,` would be correct.

Yes, I think we should just use `(uint32_t)kClockFreqHiSpeedPeripheralHz / 2`.

Reply:
done